rotkehlchen/data/importer.py

```diff
@@ -784,6 +784,74 @@ def import_blockfi_trades_csv(self, filepath: Path) -> Tuple[bool, str]:
                     return False, str(e)
         return True, ''
 
+    def _consume_binance_purchases(self, csv_row: Dict[str, Any]) -> None:
+        """
+        Consume the file containing only purchases by credit/debit card from Binance.
+        """
+        # TODO perhaps put the 'Completed' into a variable as it may change
+        if csv_row['Status'] != 'Completed':
+            log.debug(f'Ignoring not completed Binance purchase. {csv_row}')
+            return
+
+        timestamp = deserialize_timestamp_from_date(
+            date=csv_row['Date(UTC)'],
+            formatstr='%Y-%m-%d %H:%M:%S',
+            location='Binance',
+        )
+
+        amount, base_asset = csv_row['Final Amount'].split(' ')
+        base_asset = symbol_to_asset_or_token(base_asset)
+        amount = deserialize_asset_amount(amount)
+        quote_amount, quote_asset = csv_row['Amount'].split(' ')
+        quote_asset = symbol_to_asset_or_token(quote_asset)
+        quote_amount = deserialize_asset_amount(quote_amount)
+        fee, fee_currency = csv_row['Fees'].split(' ')
+        fee = deserialize_fee(fee)
+        fee_currency = symbol_to_asset_or_token(fee_currency)
+        rate = Price((quote_amount + fee) / amount)
+        trade = Trade(
+            timestamp=timestamp,
+            location=Location.BINANCE,
+            base_asset=base_asset,
+            quote_asset=quote_asset,
+            trade_type=TradeType.BUY,
+            amount=amount,
+            rate=rate,
+            fee=fee,
+            fee_currency=fee_currency,
+            link='',
```

Comment:
So I know the "link" name is terrible, but actually this field is used as a unique identifier of trades from exchanges. All exchange identifiers go to this field. From your mock data I see there is a `tranaction_id`. Can you put this here? It would help in uniqueness of the trade and rejection by rotki if you try to re-import it.